autotest/test_prms_sro.py

```diff
@@ -0,0 +1,40 @@
+import os
+import sys
+
+import pytest
+
+cwd = os.getcwd()
+if cwd.endswith("autotest"):
+    sys.path.append("..")
+    rel_path = ".."
+elif cwd.endswith("pynhm"):
+    sys.path.append(".")
+    rel_path = "."
+
+import pynhm
+
+base_path = os.path.join(rel_path, "test_data")
+
+parameter_file_dict = {
+    "box_01": os.path.join(base_path, "box_01", "box_01.params"),
+}
+
+
+@pytest.mark.parametrize("domain_key", parameter_file_dict.keys())
+def test_prms_surface_runoff(domain_key):
+    parameter_file = parameter_file_dict[domain_key]
+    print(f"parsing...'{parameter_file}'")
+
+    p_parameters = pynhm.utils.PrmsParameters(parameter_file)
```

Comment:
if this test set grows, you might want to make a parameters fixture as in test_preprocess_cbh